.github/workflows/build.yml

```diff
@@ -95,26 +95,28 @@ jobs:
       uses: actions/checkout@v2
     - name: Initialize submodules
       run: |
+        cp .gitmodules_https .gitmodules
         submodules/initialize.sh
     - name: Download prerequisites
       run: |
         wget -r -np -nd -nv -R "index.html*" -e robots=off -P build/win_x86 https://geomop.nti.tul.cz/prerequisites/
         cd build/win_x86
-        unzip gmsh-3.0.5-Windows64.zip
-        mv gmsh-3.0.5-Windows gmsh
+        unzip gmsh-4.9.1-Windows64.zip
```

Comment:
- get just Python installer
- drop dependency on gmsh, we should use GSMH comming with bgem package

Reply:
Hotovo.